### PR TITLE
Update Get-TlsCipherSuite.md

### PR DESCRIPTION
**Pull request by doctordns**
Updated the language in the synopsis.

Updated the cmdlet's Name parameter description and noted that the comparison is case sensitive.

Noted that `-Name AES` is not the same as `-Name aes` are different.

### CODE REVIEW

File: docset/winserver2019-ps/tls/Get-TlsCipherSuite.md [hunk 1]
```diff
@@ -99,15 +99,17 @@ Name                  : TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
 Protocols             : {771, 65277}
 ```
 
-This command gets all the cipher suites that have names that contain the string AES.
+This command gets all the cipher suites that have names that contain the string `AES`.
+Note that the name match is case sensitive and this command returns no output for the name `aes`.
 The output includes a field for the TLS/SSL protocols supported by the cipher. See [Cipher Suites in TLS/SSL (Schannel SSP)](https://docs.microsoft.com/windows/desktop/secauthn/cipher-suites-in-schannel) for more information.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/tls/Get-TlsCipherSuite.md:104]:
```suggestion
The output includes a field for the TLS/SSL protocols supported by the cipher. See [Cipher Suites in TLS/SSL (Schannel SSP)](/windows/desktop/secauthn/cipher-suites-in-schannel) for more information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - Many thanks @doctordns, the copy is fine but, although not related to your PR, could I please ask you to also commit my suggested change to change the link to a relative link? The relative link may not work in preview mode but it will work when the changes go live.

**Comment by doctordns** [comment]
Have accepted the suggestions - thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
Thank you @doctordns.

The same update should apply to W2016 and W2022.

**Comment by doctordns** [comment]
> Thank you @doctordns.
> 
> The same update should apply to W2016 and W2022.

How do I do that??

File: docset/winserver2019-ps/tls/Get-TlsCipherSuite.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ title: Get-TlsCipherSuite
 # Get-TlsCipherSuite
 
 ## SYNOPSIS
-Gets the list of cipher suites for TLS for a computer.
+Gets the cipher suites for TLS for a computer.
```
Comment (tseward) [inline docset/winserver2019-ps/tls/Get-TlsCipherSuite.md:14]: Could this be rephrased to "Gets the list of TLS cipher suites available on the computer."?

**Review by tseward — CHANGES_REQUESTED** [review]

File: docset/winserver2019-ps/tls/Get-TlsCipherSuite.md (hunk 2 above)
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/tls/Get-TlsCipherSuite.md:14]:
```suggestion
Gets the list of TLS cipher suites available on the computer.
```

Comment (JohanFreelancer9) [reply docset/winserver2019-ps/tls/Get-TlsCipherSuite.md:14]: @doctordns - please commit the suggested change as per @tseward's request. Many thanks.

Comment (doctordns) [reply docset/winserver2019-ps/tls/Get-TlsCipherSuite.md:14]: My thinking here is that the command returns a collection of objects. A list or table is the result of default formatting and not the command.. So the command gets the cypher suites. 

At least that is why I made the change.

**Comment by get-itips** [comment]
> > Thank you @doctordns.
> > The same update should apply to W2016 and W2022.
> 
> How do I do that??

find the articles for those versions in your own patch branch and make the edits there
https://github.com/doctordns/windows-powershell-docs/tree/patch-3

**Review by get-itips — CHANGES_REQUESTED** [review]
as per comment

**Comment by doctordns** [comment]
I _think_ I have made all the changes to all three docs - and made them correctly and consistently.

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: docset/winserver2022-ps/tls/Get-TlsCipherSuite.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ title: Get-TlsCipherSuite
 # Get-TlsCipherSuite
 
 ## SYNOPSIS
-Gets the list of cipher suites for TLS for a computer.
+2022
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/tls/Get-TlsCipherSuite.md:14]:
```suggestion
Gets the TLS cipher suites for a computer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@doctordns - Could you please commit my suggested change? I don't know where the 2022 came from.

**Comment by e0i** [comment]
@doctordns Thank you for this contribution and following up with the reviewers’ suggestions. The PR is good to go once the only remaining copy editing suggestion has been applied. Thanks.

**Comment by doctordns** [comment]
Not sure where the 2022 came from - but accepted.

Looks like this is ready to go.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by e0i — APPROVED** [review]

**Pull request merged**